.github/workflows/release.yml

```diff
@@ -55,6 +55,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Build Slack message
+        if: steps.changesets.outputs.published == 'true'
+        run: echo "UPDATED_PACKAGES=$(${{ steps.changesets.output.publishedPackages }} | jq -r '[group_by(.name) | .[] | \" - \"+.[].name+\"@\"+.[].version] | join(\"\n\")')" >> $GITHUB_ENV
```

Comment:
This formats the output of the publish step to get us a nice list of package versions that were published.

Reply:
oh so cool!

Reply:
The only thing I'm uncertain about is if `${{ steps.changesets.output.publishedPackages }}` needs to be surrounded with single quotes or not. We will have to see what happens when it runs.